### ISSUE TITLE
small fix typos

### ISSUE THREAD

**Issue by lcerrato** [PerseusDL/lexica#21]
lat.ls.perseus-eng1.xml

user report
Do you make a note of errors on the Perseus site?  I noticed the following error in the online Lewis & Short dictionary.  Under the word 'amictus' there is an incorrect reference to Tibullus.  The quotation is from Tib. 1.8.13, not Tib. 1.9.13  
"II. Meton., abstr. pro concr., the garment itself that is thrown about or on, any clothing, a mantle, cloak, etc.: “quam (statuam) esse ejusdem, status, amictus, anulus, imago ipsa declarat,” Cic. Att. 6, 1, 17: “frustra jam vestes, frustra mutatur amictus,” Tib. 1, 9, 13"

**Comment by lcerrato** [comment]
user report

Hello,  I do not know if this is where we are supposed to send typographical errors we find (if we are supposed to at all!), but I noticed in the "confido" entry in Lewis & Short that confisus sum has 'i' marked as short, whereas I am pretty sure (and Elem. Lewis) says is it long.

**Comment by lcerrato** [comment]
user report

Hi, webmaster.
  In LS&J, word φίαλη, paragraph 2, says "fiat bowl" but must say "flat bowl" with "l" not "i".  
You are wellcome. Best regards.

**Comment by lcerrato** [comment]
user report

Another one.
  http://www.perseus.tufts.edu/hopper/morph?l=eiserxomai&la=greek#lexicon  
This time,the Vietnamese(?) who digitized this part of LSJ  saw a lambda in the text and typed it as an iota.   The correct future of εἰσέρχομαι is εἰσελεύσομαι.

**Comment by lcerrato** [comment]
user report

 I looked up a word, ναῦς,νεώς,ἡ a ship  The result was:  — Att. d<*><*><*> νεω<*>ν （νηω<*>ν i  
http://www.perseus.tufts.edu/hopper/text?doc=Perseus%3Atext%3A1999.04.0057%3Aentry%3Dnau%3Ds

  This should be; ---Att.declens.ναῦς,νεώς,νηΐ,ναῦν,dual gen.and dat.νεοῖν....

**Comment by lcerrato** [comment]
user report

 http://www.perseus.tufts.edu/hopper/morph?l=a%29ne%2Fw|gon&la=greek#lexicon  
Here Perseus gives as imperfect : impf. A.“ἀνἔῳγον”  
This is wrong.  
The correct spelling is ἀνέῳγον

**Comment by lcerrato** [comment]
user report 
Here is another typo.  
Your LSJ is:  ἀνατα^ράσσω , Att. ἀνά-ττω ,
  My Intermedeate is:  
Attic. -ττω 
   http://www.perseus.tufts.edu/hopper/morph?l=anatarassw&la=greek#lexicon
  This verb is used only once in Xenophone's Anabasis.

**Comment by lcerrato** [comment]
user report 6/3/16

I found a typo in the entry "naucum" of Lewis & Short's "A Latin Dictionary": in the first line you can read "cf. hugae" when it should be "cf. nugae".

http://www.perseus.tufts.edu/hopper/text?doc=Perseus:text:1999.04.0059:entry=naucum

I've always been impressed by the accuracy of the transcription of L&S's dictionary, and I'm happy to say this is the only typo I've managed to come across after months and months of use...

I didn't know who else to e-mail regarding this.

**Comment by lcerrato** [comment]
re hugae v nudge, user is correct: the page is smudged.

**Comment by telemachus** [comment]
These changes don't seem to have been made in the Lewis and Short xml file here nor in the online dictionary. They're very minor, but should be easy to do. Would a pull request help? (I don't know the process here, and there's no note for contributors.)

**Comment by lcerrato** [comment]
Hi, 
Thanks for your interest.
These files have no relationship to those visible in Perseus, as noted in the wiki:
https://github.com/PerseusDL/lexica/wiki#making-changes-and-reporting-errors

Changes noted here will have been made in the related branch, but these have not necessarily been merged into the master file (yet).

These files are not part of the current text conversion workflow, so there are corrections logged elsewhere. This work is being reviewed and integrated into these files gradually.
--Lisa
